api/provider/aws/certificates.go

```diff
@@ -218,24 +224,30 @@ func resolveCertificateChain(body string) (string, error) {
 		return "", nil
 	}
 
-	cmd := exec.Command("cfssl", "bundle", "-cert", "-")
+	// return if this is a cloudflare origin cert
+	ou := crt.Issuer.OrganizationalUnit
+	if len(ou) == 1 && ou[0] == "CloudFlare Origin SSL Certificate Authority" {
```

Comment:
Instead of hard coding this we can probably just move forward with no chain if we can't resolve one. And could print the cfssl output as a warning but maybe not necessary.